modules/ROOT/pages/sysconfig-configure-wireguard.adoc

```diff
@@ -239,8 +285,18 @@ storage:
           AllowedIPs = 192.168.71.0/24,fdc9:3c6b:21c7:e6bd::/64
 systemd:
   units:
-    - name: wg-quick@wg0.service
+    - name: import-wireguard-config.service
```

Comment:
Let's keep the `@wg0` in the unit name to use it in the unit (I think it's `%u`?).

Reply:
Nice idea ! added.